src/main/java/org/example/GeometrijskiLik.java

```diff
@@ -2,4 +2,26 @@
 
 public class GeometrijskiLik {
 
+    GeometrijskiLik(String boja){
```

Comment:
Konstruktorima stavi da je access modifier public. Ovako bez access modifiera, access je package-private, što znači da se konstruktori vide samo iz paketa u kojem se nalazi ova klasa. Nama je sve u istom paketu, pa nema veze, ali da imamo više paketa, ne bi mogao pozvati konstruktore ove klase iz drugog paketa.